modules/permutiveRtdProvider.js

```diff
@@ -260,6 +272,34 @@ function readSegments (key) {
   }
 }
 
+const unknownIabSegmentId = '_unknown_'
+
+/**
+ * Functions to apply to ORT2B2 `user.data` objects.
+ * Each function should return an a new object containing a `name`, (optional) `ext` and `segment`
+ * properties. The result of the each transformation defined here will be appended to the array
+ * under `user.data` in the bid request.
+ */
+const ortb2UserDataTransformations = {
+  iabAudienceTaxonomy11: (userData, config) => ({
+    name: userData.name,
+    ext: { segtax: '4' },
```

Comment:
If you're making pubs define all the transformations, no reason you can't let this be a param...